services/wfm/src/main/java/org/openkilda/wfm/topology/flow/service/FlowService.java

```diff
@@ -1539,6 +1539,17 @@ private void unwrapBaseFaisafeException(FailsafeException e) throws UnroutableFl
         return swapFlows(currentFirstFlow, existingFirstFlow, currentSecondFlow, existingSecondFlow, sender);
     }
 
+    /**
+     * Fetch flow and provide update on it's periodic pings.
+     * @param flowId flow id
+     * @param sender interface
+     */
+    public void notifyOnPeriodicPingChanges(String flowId,
+                                            FlowCommandSender sender) {
+        Flow flow = flowRepository.findById(flowId).get();
```

Comment:
what is there is no such flow?

Reply:
it will be cleaned up by periodic pings cache flush in ping topo with a one minute delay